website/src/configs/docsList.json

```diff
@@ -94,6 +94,12 @@
     "title": "Svelte / SvelteKit",
     "path": "/docs/guides/sveltejs"
   },
+  {
+    "key": "astro",
```

Comment:
Oops, I should've caught this last review, but we should also keep the order alphabetical. Underneath Angular would be great!